Dockerfile

```diff
@@ -0,0 +1,72 @@
+ARG REGISTRY_PREFIX=''
+ARG CODENAME=bionic
+
+FROM ${REGISTRY_PREFIX}ubuntu:${CODENAME} as builder
+
+RUN set -x \
+	&& apt update \
+	&& apt upgrade -y \
+	&& apt install --yes --no-install-recommends \
+		build-essential \
+		autotools-dev \
+		automake \
+		pkg-config \
+                libltdl-dev \
+		ca-certificates \
+		curl \
+		libqt4-dev \
+		libqt4-sql-sqlite \
+		x11-apps 
+
+ARG PARALLELMFLAGS=-j2
+
+ARG BUILD_DIR=/tmp/build
+
+ARG DUMB_INIT_VERSION=1.2.2
```

Comment:
dumb-init provides a basic init process.
Since the init process does some special handling (signals, child processes), it should be avoided to use an application directly as docker entrypoint (see below).